app/views/tools/_petition.html.erb

```diff
@@ -124,10 +124,7 @@
               <% unless @require_location %>
                 <%= render 'tools/petition_location', f: f %>
               <% end %>
-
-              <%= render "tools/email_signup_radios" unless user_signed_in? %>
```

Comment:
Non-blocking: Is it intentional that the `unless user_signed_in?` condition is lost?